nemo_text_processing/text_normalization/ja/data/numbers/digit_alt.tsv

```diff
@@ -0,0 +1,8 @@
+2	二
```

Comment:
Not sure if this was already addressed, but if it's generally the same set of mappings, recycling `digit.tsv` sans the mapping for `1` would help reduce the overall size of the grammar.

Reply:
Updated the gramamar. Let me remove this data file.